toolkits/landcover/ee_repo_sync.py

```diff
@@ -69,10 +75,12 @@ def search_replace(start_path, ext, original, replacement):
     new_require_path += target_path + '/'
 
 # Clone or pull repo to home dir
-if not os.path.isdir(clone_path):
-    subprocess.call(['git', 'clone', repo_url, clone_path])
+if os.path.isdir(clone_path):
+    print('%s exists. Pulling latest from remote.' % clone_path)
+    subprocess.call(['git', '-C', clone_path, 'pull', '--quiet'])
 else:
-    subprocess.call(['git', '-C', clone_path, 'pull'])
+    print('%s not found. Cloning remote.' % clone_path)
+    subprocess.call(['git', 'clone', repo_url, clone_path])
 
 subprocess.call(['mkdir', '-p', write_path])
```

Comment:
What should happen if the write_path already exists?

Reply:
Nothing, `mkdir -p` is idempotent.